templates/default.html

```diff
@@ -2,10 +2,16 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Resin.io Documentation</title>
+    <title>{{ title }} - Resin.io Documentation</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="shortcut icon" type="image/x-icon" href="/favicon.png" />
 
+    <meta property="fb:app_id" content="221218511385682" />
```

Comment:
I see. Nope, we don't do it here because otherwise we'd generate tons of
PRs :)
Just to eliminate extra work here I suggest you close the original PR and
we keep working with the more recent.
But from now on no need to open new PRs — you just keep pushing new commits
to the same branch.
When the PR is good to merge it's good to rebase the branch and squash the
commits where applicable (then force-push it).

On Wed, Jul 27, 2016 at 2:58 PM, Gergely Imreh <notifications@github.com>
wrote:

> In templates/default.html
> <https://github.com/resin-io/docs/pull/260#discussion_r72426311>:
>
> >      <meta name="viewport" content="width=device-width, initial-scale=1.0">
> >      <link rel="shortcut icon" type="image/x-icon" href="/favicon.png" />
> >
> > +    <meta property="fb:app_id" content="221218511385682" />
>
> Hm, good question, habits, I guess? Thinking about it, it is a holdover
> from when I was contributing to Buildroot a bit, or was it some other
> project? Either way, most other project I've seen that if the PR needs
> changes, it comes in as another PR, not changes to the existing one (which
> is closed without merging) - ie. don't add commits to change things in PRs
> that needs fixing, and don't force push changes to existing commits.
>
> I'm happy to do in any pattern that we use, haven't seen such info yet.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/resin-io/docs/pull/260/files/6cb9539aa5363190f837449d9b617f2ad9e7f83a#r72426311>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgGCCxq1gsheNUV2Mee2tesneoADwSGks5qZ0gCgaJpZM4JV_Uc>
> .
>



-- 
Eugene Mirotin
Senior Frontend Engineer
site: Resin.io <https://resin.io/>, twitter: @resin_io
<https://twitter.com/resin_io>